client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -150,19 +170,42 @@ const CheckoutThankYou = React.createClass( {
 	},
 
 	render() {
-		const classes = classNames( 'checkout-thank-you', {
-			'is-placeholder': ! this.isDataLoaded()
-		} );
-
 		let purchases = null,
-			wasJetpackPlanPurchased = false;
+			wasJetpackPlanPurchased = false,
+			wasOnlyDotcomPlanPurchased = false;
 		if ( this.isDataLoaded() && ! this.isGenericReceipt() ) {
 			purchases = getPurchases( this.props );
 			wasJetpackPlanPurchased = purchases.some( isJetpackPlan );
+			wasOnlyDotcomPlanPurchased = purchases.every( isPlan );
+		}
+
+		const userCreatedMoment = moment( this.props.user.date );
```

Comment:
I think that is fine in this case. If it is `null` for some reason, we will treat a user as new